pkg/v1/providers/tests/unit/avi_test.go

```diff
@@ -0,0 +1,261 @@
+	}))
+	assertNotFound(matchers.FindDocsMatchingYAMLPath(output, map[string]string{
+		"$.kind":          "ClusterRoleBinding",
+		"$.metadata.name": "ako-operator-bootstrap-rolebinding",
+	}))
+	assertNotFound(matchers.FindDocsMatchingYAMLPath(output, map[string]string{
+		"$.kind":          "StatefulSet",
+		"$.metadata.name": "ako-operator-controller-manager",
+	}))
+	assertNotFound(matchers.FindDocsMatchingYAMLPath(output, map[string]string{
+		"$.kind":          "Secret",
+		"$.metadata.name": AviAdminCredentialName,
+	}))
+	assertNotFound(matchers.FindDocsMatchingYAMLPath(output, map[string]string{
+		"$.kind":          "Secret",
+		"$.metadata.name": AviCaName,
+	}))
+}
+
+func AllAVIRelatedObjects(output string) {
+	assertFoundOne(matchers.FindDocsMatchingYAMLPath(output, map[string]string{
+		"$.kind":          "Secret",
+		"$.metadata.name": "avi-secret",
+	}))
+	assertFoundOne(matchers.FindDocsMatchingYAMLPath(output, map[string]string{
+		"$.kind":          "ServiceAccount",
+		"$.metadata.name": "ako-sa",
+	}))
+	assertFoundOne(matchers.FindDocsMatchingYAMLPath(output, map[string]string{
+		"$.kind":          "ConfigMap",
+		"$.metadata.name": "avi-k8s-config",
+	}))
+	assertFoundOne(matchers.FindDocsMatchingYAMLPath(output, map[string]string{
+		"$.kind":          "ClusterRole",
+		"$.metadata.name": "ako-cr",
+	}))
+	assertFoundOne(matchers.FindDocsMatchingYAMLPath(output, map[string]string{
+		"$.kind":          "ClusterRoleBinding",
+		"$.metadata.name": "ako-crb",
+	}))
+	assertFoundOne(matchers.FindDocsMatchingYAMLPath(output, map[string]string{
+		"$.kind":          "StatefulSet",
+		"$.metadata.name": "ako",
+	}))
+	assertFoundOne(matchers.FindDocsMatchingYAMLPath(output, map[string]string{
+		"$.kind":          "Service",
+		"$.metadata.name": "ako-operator-controller-manager-metrics-service",
+	}))
+	assertFoundOne(matchers.FindDocsMatchingYAMLPath(output, map[string]string{
+		"$.kind":          "ClusterRole",
+		"$.metadata.name": "ako-operator-manager-bootstrap",
+	}))
+	assertFoundOne(matchers.FindDocsMatchingYAMLPath(output, map[string]string{
+		"$.kind":          "ClusterRoleBinding",
+		"$.metadata.name": "ako-operator-bootstrap-rolebinding",
+	}))
+	assertFoundOne(matchers.FindDocsMatchingYAMLPath(output, map[string]string{
+		"$.kind":          "Deployment",
+		"$.metadata.name": "ako-operator-controller-manager",
+	}))
+	assertFoundOne(matchers.FindDocsMatchingYAMLPath(output, map[string]string{
+		"$.kind":          "Secret",
+		"$.metadata.name": AviAdminCredentialName,
+	}))
+	assertFoundOne(matchers.FindDocsMatchingYAMLPath(output, map[string]string{
+		"$.kind":          "Secret",
+		"$.metadata.name": AviCaName,
+	}))
+}
+
+var _ = Describe("AKO-operator Ytt Templating", func() {
+
+	var paths []string
+	var baseVal yttValues
+
+	BeforeEach(func() {
+		paths = []string{
+			filepath.Join(yamlRoot, "config_default.yaml"),
+			filepath.Join("./fixtures/tkr-bom-v1.21.1.yaml"),
```

Comment:
Can we use newer version 1.5?

Reply:
I only saw these two fixtures, only for 1.4.0. there is a need to update it. Will do it in a separate PR